src/PKSim/Program.cs

```diff
@@ -20,7 +20,7 @@ private static void Main(string[] args)
          Application.EnableVisualStyles();
          Application.SetCompatibleTextRenderingDefault(false);
 
-         //   WindowsFormsSettings.SetDPIAware();
+         WindowsFormsSettings.SetDPIAware();
```

Comment:
on my laptop, PKSim is not usable without this setting turned on (everyhting is blurry).